udf/map_mode_last.sql

```diff
@@ -2,3 +2,28 @@
 CREATE OR REPLACE FUNCTION udf.map_mode_last(entries ANY TYPE) AS (
   mozfun.map.mode_last(entries)
```

Comment:
@asiOvOtus if this is what you mean by stubbing (shimming) the functions, then I think this is a good pattern. `mozfun` should be the location where all of the logic is implemented, with these functions getting removed sometime in the future.